
string name
shape_msgs/Mesh
float mass
bool from_faces
bool randomize_constraints
uint8 bending_contrainst_distance
uint8 piterations
geometry_msgs/Vector3 scale
geometry_msgs/Pose pose
